lib/tools/adb-commands.js

```diff
@@ -47,8 +47,8 @@ methods.initZipAlign = async function () {
 /**
  * Retrieve the API level of the device under test.
  *
- * @return (string) The API level as integer number, for example 21 for
- * Android Lollipop. The result of this method is cached, so all the further
+ * @return {string} The API level as integer number, for example 21 for
```

Comment:
Maybe... Maybe it even makes sense to cache this value.
But due to some historical reasons unknown to me this method is implemented this way.